src/api/SFA.DAS.Provider.Events.Api/Web.config.orig

```diff
@@ -9,7 +9,6 @@
   </configSections>
   <appSettings>
     <add key="LogLevel" value="Debug"/>
-    <add key="LoggingConnectionString" value="server=.;database=PaymentsEventApi;trusted_connection=true;" />
 <<<<<<< HEAD

```

Comment:
this file should not be checked in as it gets created when there is a merge conflict, please delete this file

Reply:
File removed